src/mockDataset.js

```diff
@@ -1,40 +1,395 @@
 export const monsters = {
-  "data": {
-    "queryMonsters": [
-      {
-        "index": "adult-gold-dragon",
-        "name": "Adult Gold Dragon",
-        "size": "HUGE",
-        "type": "DRAGON",
-        "alignment": "lawful good",
-        "challenge_rating": 17
+  "queryMonsters": [
```

Comment:
All this looks so awesome, I wish I understood a bit more to give better feedback. The endpoint for this should be live, so you can hook it in and please let us know of any changes we can make to the endpoint to make your life easier!